lua/zenbones/shipwright/runners/windows_terminal.lua

```diff
@@ -0,0 +1,41 @@
+local template = [[{
+"background": "$bg",
+"black": "$black",
+"blue": "$blue",
+"brightBlack": "$bright_black",
+"brightBlue": "$bright_blue",
+"brightCyan": "$bright_cyan",
+"brightGreen": "$bright_green",
+"brightPurple": "$bright_magenta",
+"brightRed": "$bright_red",
+"brightWhite": "$bright_white",
+"brightYellow": "$bright_yellow",
+"cursorColor": "$cursor_bg",
+"cyan": "$cyan",
+"foreground": "$fg",
+"green": "$green",
+"name": "zenbonse",
```

Comment:
Name should be the colorscheme name.

Reply:
we have any variable for this $???

Reply:
Yes, it just `name`. Checkout the kitty runner where it uses the name. It should pretty much the same.

Reply:
- i read the shipwright.nvim [code](https://github.com/rktjmp/shipwright.nvim/blob/a3fee9725346eba3eacba36989c906a550a32ae7/lua/shipwright/transform/contrib/kitty.lua?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L168)
```lua
  local replaced = helpers.split_newlines(helpers.apply_template(template, colors))
  local kept = {}
  for _, line in ipairs(replaced) do
    if not string.match(line, "%$") then
      table.insert(kept, line)
    end
  end

  return kept
 ```
 - i think this code will convert $name, i try ~40m write it in windows terminal runner but not worked 😅

Reply:
I meant the kitty runner in zenbones. https://github.com/mcchrish/zenbones.nvim/blob/99da53ada1715f1f5758b936f03ce5dab0dfe5cc/lua/zenbones/shipwright/runners/kitty.lua#L5-L19